### PR TITLE
Exporting the loss ratio in the asset loss curves

### PR DESCRIPTION
**Pull request by micheles**
As requested by @CatalinaYepes . Here is an example for the output loss_curves-mean.csv:
```
#,,,,"kind='mean', risk_investigation_time=1.0"
asset_id,loss_type,loss_value,loss_ratio,period
a_0,structural,0.00000E+00,0.00000E+00,5
a_0,structural,0.00000E+00,0.00000E+00,10
a_0,structural,0.00000E+00,0.00000E+00,25
a_0,structural,0.00000E+00,0.00000E+00,50
a_0,structural,7.62277E+03,2.53270E-03,100
a_0,structural,1.84955E+04,6.14523E-03,250
a_0,structural,2.90040E+04,9.63672E-03,500
a_0,structural,5.25483E+04,1.74594E-02,1000
a_0,structural,8.16276E+04,2.71212E-02,1500
a_0,structural,1.75810E+05,5.84137E-02,2500
...
a_14,structural,0.00000E+00,0.00000E+00,5
a_14,structural,0.00000E+00,0.00000E+00,10
a_14,structural,0.00000E+00,0.00000E+00,25
a_14,structural,0.00000E+00,0.00000E+00,50
a_14,structural,1.02342E+02,1.42497E-04,100
a_14,structural,9.24486E+02,1.28722E-03,250
a_14,structural,3.07955E+03,4.28784E-03,500
a_14,structural,9.09675E+03,1.26660E-02,1000
a_14,structural,2.09473E+04,2.91662E-02,1500
a_14,structural,5.63643E+04,7.84794E-02,2500
```
Part of https://github.com/gem/oq-engine/issues/5738. This is not enough to solve the bug, which is caused by a wrong asset <-> asset_ref association.

### CODE REVIEW

File: openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-000.csv [hunk 1]
```diff
@@ -1,158 +1,158 @@
...
-a1,structural,9.07514E+07,1.48324E-09
-a1,structural,9.08030E+07,1.47494E-09
-a1,structural,9.08547E+07,1.46669E-09
-a1,structural,9.09063E+07,1.45848E-09
-a1,structural,9.13068E+07,1.39645E-09
-a1,structural,9.17072E+07,1.33715E-09
-a1,structural,9.21077E+07,1.28048E-09
-a1,structural,9.25082E+07,1.22629E-09
-a1,structural,9.29086E+07,1.17449E-09
+#,,,,"kind='rlz-000', risk_investigation_time=None"
```
Comment (raoanirudh) [inline openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-000.csv:1]: If `risk_investigation_time` is not explicitly specified in the job file, the engine uses `risk_investigation_time = investigation_time`

File: openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#,,,,"kind='quantile-0.85', risk_investigation_time=50.0"
+asset_id,loss_type,loss_value,loss_ratio,period
```
Comment (raoanirudh) [inline openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv:2]: `period` → `return_period` for the field names to be consistent with existing outputs.

Comment (raoanirudh) [reply openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv:2]: The `event_based_risk` and `ebrisk` calculators also have an `annual_frequency_of_exceedence` field.

File: openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-000.csv (hunk 1 above)
Comment (micheles) [reply openquake/qa_tests_data/classical_risk/case_4/expected/loss_curves-000.csv:1]: Fixed

File: openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv (hunk 2 above)
Comment (micheles) [reply openquake/qa_tests_data/gmf_ebrisk/case_4/expected/loss_curves-quantile-0.85.csv]: Fixed

**Comment by micheles** [comment]
I will leave the fix to a different PR

**Pull request merged**